server/jetstream_api.go

```diff
@@ -3320,6 +3325,13 @@ func (s *Server) jsConsumerInfoRequest(sub *subscription, c *client, subject, re
 				resp.Error = jsClusterNotAvailErr
 				s.sendAPIErrResponse(ci, acc, subject, reply, string(msg), s.jsonResponse(&resp))
 			}
+			// Check here if we are a member and this is just a new consumer that does not have a leader yet.
+			if rg := ca.Group; rg.node != nil && rg.isMember(cc.meta.ID()) {
```

Comment:
Is `rg` guaranteed to not be nil? I see that we check for `rg.node != nil` but not sure if `rg` itself can be nil.

Reply:
Should not be but will add a check for saftey.

Reply:
Will reflow.